### DISCUSSION TITLE
GNPy adoptes dijkstra algorithm

### DISCUSSION THREAD

**Discussion by indayara** [Telecominfraproject/oopt-gnpy#438] [General]
**Is your feature request related to a problem? Please describe.**

I see that the GNPy adopted the Yang path model for the path computation process. 
Is it possible to integrate/apply different algorithms for the path computation process?

Thanks in advance.
Best regards,

**Comment by jktjkt** [comment]
Hi, right now this is not possible (but it should be possible to compute the path externally and send back a request which -- effectively -- has the path already set in stone). What is your use case for this feature?

(I'm also moving this to a GitHub discussion because it's not really a bugreport.)

**Comment by ojnas** [comment]
Could you elaborate a bit on what you mean by "different algorithms" and what you want to achieve? For example, do you want to:
1. use something else than Dijkstra for computing the shortest path? If so, why?
2. compute shortest path based on other metrics than distance? If so, what metrics?
3. compute multiple alternative paths?
4. ... etc.

**Reply by indayara** [reply]
Hi, 
I would like to know if I could use something else than Dijkstra for computing the shortest path.
Why?
It's just a matter of technique.
Thanks.
